tests/sentry/event_manager/grouping/test_assign_to_group.py

```diff
@@ -494,16 +477,14 @@ def test_existing_group_new_hash_exists(
     "in_transition", (True, False), ids=(" in_transition: True ", " in_transition: False ")
 )
 @pytest.mark.parametrize(
-    "mobile_config", (True, False), ids=(" mobile_config: True ", " mobile_config: False ")
+    "id_qualifies", (True, False), ids=(" id_qualifies: True ", " id_qualifies: False ")
```

Comment:
You're probably going to run into a merge conflict here, since I modified this in https://github.com/getsentry/sentry/pull/77027. (Sorry!)

Reply:
All good. This means we're working hard!! 😄